version.xml

```diff
@@ -13,8 +13,8 @@
 <version>
 	<application>pln</application>
 	<type>plugins.generic</type>
-	<release>2.0.4.1</release><!-- 2.0.4.x is for ojs v.3.3 -->
-	<date>2020-07-21</date>
+	<release>2.0.5.0</release><!-- 2.0.5.x is reserved for OJS 3.4 -->
```

Comment:
You've bumped the version [elsewhere](https://github.com/pkp/pln/commit/831f6271b1e391ccb879d16bf23b9392c022b57f) to `3.0.0.0`. I'd suggest maintaining the version outside of any single pull request. No strong preference here for whether you keep it 2.x or 3.x.

Reply:
I was in doubt between following the previous range or bumping it to 3, so this is just old and will be flagged as a conflict :)